src/HtmlGenerator/Program.cs

```diff
@@ -178,6 +185,18 @@ private static void Main(string[] args)
                 return;
             }
 
+            if (rootPath is object)
+            {
+                foreach (var project in projects)
+                {
+                    if (!Paths.IsOrContains(rootPath, project))
+                    {
+                        Log.Exception("If a /root folder is specified, it must be an ancestor folder of all specified projects.", isSevere: true);
+                        return;

```

Comment:
@KirillOsenkov I'm a bit worried because when this code executes, no text appears in the console window when debugging. I also tried `Log.Close()` before returning and it didn't make a difference.

Reply:
This is what it does:
http://sourcebrowser.azurewebsites.net/#Microsoft.SourceBrowser.Common/Log.cs,ee4ecaf77645c4cd

do you have any guesses on why this might not work? I haven't noticed this before and haven't tried to repro myself yet.

Reply:
Well, that's not what it does for starters. According to "go to decompiled sources" in VS, `Log.Write` is defined this way:

```cs
        public static void Write(string message, ConsoleColor color = ConsoleColor.Gray)
        {
            Messages.OnNext(new ConsoleMessage(message, color));
        }
 ```

Reply:
oh, sigh. My SourceBrowser source browser is stale. Someone changed logging recently, they might have regressed this.

Reply:
Sorry, that wasn't decompiled sources. Fixed the comment.

Reply:
Care filing an issue about this?

Reply:
https://github.com/KirillOsenkov/SourceBrowser/issues/165